src/plugins/navigation_embeddable/public/components/navigation_embeddable_panel_editor.tsx

```diff
@@ -227,11 +214,70 @@ const NavigationEmbeddablePanelEditor = ({
       <EuiFlyoutFooter>
         <EuiFlexGroup responsive={false} justifyContent="spaceBetween">
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty onClick={onClose} iconType="cross">
+            <EuiButtonEmpty onClick={onClose} iconType="cross" flush="left">
               {NavEmbeddableStrings.editor.getCancelButtonLabel()}
             </EuiButtonEmpty>
           </EuiFlexItem>
-          <EuiFlexItem grow={false}>{saveButtonComponent}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup>
```

Comment:
```suggestion
            <EuiFlexGroup gutterSize="m">
```

| Before | After |
|--------|--------|
| ![image](https://github.com/elastic/kibana/assets/8698078/ac247c28-caf9-417d-af64-a687b0f9aced) | ![image](https://github.com/elastic/kibana/assets/8698078/427908e7-030d-4958-bebe-ac067b785cba) |